lib/modules/@apostrophecms/schemas/src/apos/components/AposInputCheckboxes.vue

```diff
@@ -22,7 +22,7 @@
 import AposInputMixin from '../mixins/AposInputMixin';
 
 export default {
-  name: 'AposInputCheckbox',
+  name: 'AposInputCheckboxes',
```

Comment:
Did you rename this because it can display multiple checkboxes?

Reply:
If so I would support it. Unlike a module, a component is generally instantiated "once per thing," so if it's actually "once per group of things" that should be clear from the name.

Reply:
I renamed it because in `lib/modules/@apostrophecms/schemas/index.js`, there is this:

```js
self.addFieldType({
   name: 'checkboxes',
   ...
});
```

When listing the field types in `getBrowserData`, the type `checkboxes` came out. 
```js
for (const name in self.fieldTypes) {
      fields[name] = 'AposInput' + self.apos.utils.capitalizeFirst(name);
}
```

So I decided to rename the Vue component. I could have renamed the field type instead (actually, after having thought of it, I guess it would be coherent to have it singular). Let me know @boutell @abea what is best.

Reply:
I'm okay with this change.

Reply:
The field type should stay plural because it directly contains more than one checkbox, and if it had a singular name I would expect it to behave very differently, i.e. I would expect its value to be a boolean, not an array of selected checkbox names.

So the vue component's name should also be plural, to match. It directly contains more than one thing, if it had a singular name you would expect it to behave differently.

Is this the change you are OK with @abea ?